packages/react-core/src/components/Truncate/examples/Truncate.md

```diff
@@ -5,14 +5,16 @@ propComponents: [Truncate]
 ---
 
 import './TruncateExamples.css';
+import truncateStyles from '@patternfly/react-styles/css/docs/components/Truncate/examples/Truncate';
 
 ## Examples
 
 ### Default
 ```js
 import React from 'react';
 import { Truncate } from '@patternfly/react-core';
-<div className="pf-v5-c-truncate--example">
+import truncateStyles from '@patternfly/react-styles/css/docs/components/Truncate/examples/Truncate';
+<div className={truncateStyles.truncateExample}>
```

Comment:
IMO we should just call this like `<div className="truncate-example-resize">` or something, no need for the version or anything in it. It's also too similar to our BEM syntax, it may be confusing to consumers as to whether it's part of our component or not. We'll just need to then update the selector here to match (eg, `.truncate-example-resize`) https://github.com/patternfly/patternfly-react/blob/b0ce03e33126de5b99a55ff02b90ef93dcf650bf/packages/react-core/src/components/Truncate/examples/TruncateExamples.css#L1